src/fsharp/ErrorResolutionHints.fs

```diff
@@ -23,9 +23,11 @@ let IsInEditDistanceProximity userEntered suggestion =
 /// Filters predictions based on edit distance to the given unknown identifier.
 let FilterPredictions (userEntered:string) (suggestionF:ErrorLogger.Suggestions) =    
     let uppercaseText = userEntered.ToUpperInvariant()
-    suggestionF()
+    let allSuggestions = suggestionF()
+
+    if allSuggestions.Contains userEntered then [] else // some other parsing error occurred
```

Comment:
Shouldn't the IDE take care of this?  It seems like it's more of an rendering thing than something that makes sense here ?  It seems weird to me that the compiler looks for a suggestion with the text user entered in order to determine whether to filter it.

Perhaps I am missing something though